### ISSUE TITLE
UNDER CONSTRUCTION -- Use `versionIRI` as a concept prefix

### ISSUE THREAD

**Issue by plbt5** [casework/case#95]
# Background

A new issue needs to be worked through w.r.t. the proposed strategy of using the `versionIRI` as a concept prefix.

If a concept is defined with the `versionIRI` as part of its IRI, there is nothing in OWL that induces the existence of the unversioned IRI form.  E.g. `http://example.org/ontology/0.1.0/File` does not imply the existence of `http://example.org/ontology/File`.

Ultimately, the effect of this is that if a knowledge base intends to pin a version of CASE to be used, this is not easily possible. 

# Requirements

## Requirement 1
Allow to pin a particular version of CASE in a knowledge base by applying its versionIRI.

## Requirement 2
Allow to pin the most recent version of CASE in a knowledge base by default

# Risk / Benefit analysis

## Benefits
CASE allows adopters of the standard to refer to either a particular version of CASE, or to apply the default most recent version of CASE.

## Risks

The submitter is unaware of risks associated with this change

# Competencies demonstrated

#### Result 1.2

# Solution suggestion

Define within the knowledge base an all-encompassing owl:Ontology, and have it import the versionIRI of CASE.  Management issues around detection of multiple conflicting versionIRIs for a single ontology can then be handled with a SHACL rule within this proposal on Github:

[UCO should perform OWL 2 DL review with SHACL-SPARQL (Change Proposal #406)](https://github.com/ucoProject/UCO/issues/406)

So, if our frequently-exercised knowledge base IRI for example individuals, http://example.org/kb/, were to involve pinning a used version of CASE, the correct answer would be to include this in the graph pertaining to that prefix:

```
<http://example.org/kb>
    a owl:Ontology ;
    owl:imports <https://ontology.caseontology.org/case/case/0.9.0> ;
    .
```

**Comment by plbt5** [comment]
@ajnelson-nist I think that this issue has been resolved with #113 and can therefore be removed?

**Comment by ajnelson-nist** [comment]
Correct, this is resolved.
